docker-compose-example/.env

```diff
@@ -1,17 +1,17 @@
 PUID=1000
 PGID=1000
-ARM64_DEVICE=adlink
+ARM64_DEVICE=generic
 USE_DEPOT_DOWNLOADER=false
 WORLD_INDEX=0
 WORLD_NAME="Core Keeper Server"
 WORLD_SEED=0
 WORLD_MODE=0
 GAME_ID=""
-DATA_PATH="${STEAMAPPDATADIR}"
+DATA_PATH="${STEAMAPPDATADIR:-/home/steam/core-keeper-data}"
```

Comment:
We should actually not expose this env as something ppl should edit, there are very few valid reasons for someone to change this, on top of that depending on where `DATA_PATH` is pointing to it will break the server since there will be permission issues.

Reply:
Agree with you. I only keep it as it was before.